test/rss-testcase.rb

```diff
@@ -118,7 +118,7 @@ def make_channel(content=nil)
 
   <items>
     <rdf:Seq>
-#{RESOURCES.collect do |res| '<rdf:li resource="' + res + '" />' end.join("\n")}
+#{RESOURCES.collect { |res| '<rdf:li resource="' + res + '" />' }.join("\n")}
```

Comment:
```suggestion
#{RESOURCES.collect do |res| '<rdf:li resource="' + res + '" />' end.join("\n")}
```

Not in `Style/StringLiterals`.